spec/src/modules/tracker.js

```diff
@@ -20,13 +20,15 @@ const sendTimeout = 25;
 const testApiKey = process.env.TEST_API_KEY;
 const { fetch } = fetchPonyfill({ Promise });
 
-describe('ConstructorIO - Tracker', () => {
+describe.only('ConstructorIO - Tracker', () => {
   const clientVersion = 'cio-mocha';
   const waitInterval = 1000;
   let fetchSpy = null;
   let eventSpy = null;
 
-  jsdom({ url: 'http://localhost' });
+  jsdom({
+    url: 'http://localhost.test/path/name',
```

Comment:
Can we add query params here to confirm that they aren't included?

Reply:
Updated w/ query params as well as `sendReferrerWithTrackingEvents` and tests